setup.py

```diff
@@ -17,14 +15,17 @@
     # setup_requires=["setuptools_scm"],
     version="1.4.2",
     description=__doc__,
-    long_description=".. warning:: this is not meant for PyPi (yet)",
-    author="Brendan Doherty",
+    long_description=(
+        "A python package that powers the github action named cpp-linter-action. "
+        + f"See `the github repository README <{REPO}#readme>`_ for full details."
+    ),
+    author="Brendan Doherty, Peter Shen",
```

Comment:
```suggestion
    author="Brendan Doherty",
```

Reply:
You don't want your name on this?! I didn't add you email to `author_email` field.

Reply:
You are the author of the Python code, I may be a future contributor, shame, but thank you!

Reply:
I consider feedback, review, and testing as contributions.. You deserve it!